core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2374,6 +2375,12 @@ class KafkaApis(val requestChannel: RequestChannel,
         val responseString =
           if (RequestChannel.isRequestLoggingEnabled) Some(response.toString(request.context.apiVersion))
           else None
+        try {
+          auditor.record(request, response)
+        } catch {
+          case e: Throwable => error(s"Auditor failed to record $request and $response", e)
```

Comment:
This log line will print out the whole buffer of the request. Is this what you want?

Reply:
This is a good point. Logging buffer content for produce request and fetch response is not good. I will change it to log some reasonable information. Thanks